virttest/utils_test/libvirt.py

```diff
@@ -135,6 +158,28 @@ def create_bridge_xml(self):
         ip = utils_net.get_ip_address_by_interface(iface)
         return ip, net_xml
 
+    def create_nat_xml(self):
+        """
+        Create XML for a nat network.
+        """
+        address = self.kwargs.get('address')
+        if not address:
+            raise exceptions.TestError('Create nat need address be set.')
```

Comment:
Is this better if you refine the error message as below:
'address' is required to create nat network xml.